tests/integration/kubernetes/runtimeclass_workloads/cryptsetup.exp

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2025 Microsoft Corporation
+# SPDX-License-Identifier: Apache-2.0
+set timeout 60
+
+set POD_ID [lindex $argv 0]
+set DM_DEVICE [lindex $argv 1]
+
+spawn /opt/kata/bin/kata-runtime exec $POD_ID
+expect "# "
+send "cryptsetup status $DM_DEVICE\n"
```

Comment:
is there any way to run this from within the container (for a test container, may be OK to assign capabilities/privileges with a comment that this is only required for the test case)